v2/internal/frontend/desktop/windows/window.go

```diff
@@ -32,15 +36,10 @@ func NewWindow(parent winc.Controller, appoptions *options.App) *Window {
 	}
 
 	var dwStyle = w32.WS_OVERLAPPEDWINDOW
-	if appoptions.Frameless {
-		dwStyle = w32.WS_POPUP
-		if winoptions := appoptions.Windows; winoptions != nil && winoptions.EnableFramelessBorder {
-			dwStyle |= w32.WS_BORDER
-		}
-	}
 
 	winc.RegClassOnlyOnce("wailsWindow")
 	result.SetHandle(winc.CreateWindow("wailsWindow", parent, uint(exStyle), uint(dwStyle)))
+	winc.RegMsgHandler(result)
```

Comment:
What do you think we should do about this PR that also moves this call? https://github.com/wailsapp/wails/pull/1082/files

Reply:
AFAIK it was only moved to prevent a `nil` pointer dereferencing when `notifyParentWindowPositionChanged` wasn't set. Maybe the best would be to check for `nil` before calling the `notifyParentWindowPositionChanged` or adding `EventMangers` on `ControlBase` in winc for `OnMove` like the other Events, e.g. `OnSize`